Dynatrace/dynatrace-operator
# Adds correct labels to topologySpreadConstraints of webhook

# Description

Currently the matchLabels for the webhook are hardcoded and static. We should use the same label placeholder as the webhook pod has.

## How can this be tested?
Deploy the operator via helm -> check if topologySpreadConstraints have correct matchLabels and match with the Pods labels


## Checklist
